index.js

```diff
@@ -206,7 +206,9 @@ FSWatcher.prototype._emit = function(event, path, val1, val2, val3) {
     this.options.alwaysStat && val1 === undefined &&
     (event === 'add' || event === 'addDir' || event === 'change')
   ) {
-    fs.stat(path, function(error, stats) {
+    var fullPath = path;
```

Comment:
Ok, now that the tests are passing, a bit of code style feedback.

Would prefer this condensed to

```js
var fullPath = this.options.cwd ? sysPath.join(this.options.cwd, path) : path;
```